docs/antctl.md

```diff
@@ -666,3 +667,29 @@ worker1 172.18.0.4 Alive
 worker2 172.18.0.3 Alive 
 worker3 172.18.0.2 Dead
 ```
+
+### Upgrade existing resources
+
+Antctl supports upgrading existing resources of CRDs managed by Antrea to the latest version. The related sub commands
```

Comment:
Antctl -> antctl

Could you also change other "Antctl" in this doc?

Reply:
sub commands -> sub-commands

Reply:
Done